src/factories/Container.ts

```diff
@@ -0,0 +1,93 @@
+module n3Charts.Factory {
+  'use strict';
+
+  export interface IDimension {
+    width: number;
+    height: number;
+    innerWidth: number;
```

Comment:
If `innerWidth` and `innerHeight` can be computed from the other properties, they're probably redundant and will need to be maintained artificially, don't you think so ?